opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -311,3 +311,57 @@ def _convert_aggregation_temporality(
             aggregation_temporality=aggregation_temporality,
         )
     return None
+
+
+class _AggregationFactory(ABC):
+    @abstractmethod
+    def create_aggregation(self) -> Aggregation:
+        """Creates an aggregation"""
+
+
+class ExplicitBucketHistogramAggregationFactory(_AggregationFactory):
+    def __init__(
+        self,
+        boundaries: Sequence[float] = (
+            0.0,
+            5.0,
+            10.0,
+            25.0,
+            50.0,
+            75.0,
+            100.0,
+            250.0,
+            500.0,
+            1000.0,
+        ),
+        record_min_max: bool = True,
+    ) -> None:
+        self._boundaries = boundaries
+        self._record_min_max = record_min_max
+
+    def create_aggregation(self) -> Aggregation:
```

Comment:
```suggestion
    def _create_aggregation(self) -> Aggregation:
```
To minimize the API. Also, this method should be private because the user should not have access to it since it would be returning an instance of a private class, like `_SumAggregation`.